app/models/host/base.rb

```diff
@@ -196,6 +196,13 @@ def set_interfaces(parser)
       if !self.managed? && self.primary_interface.mac.blank? && self.primary_interface.identifier.blank?
         identifier, values = parser.suggested_primary_interface(self)
         self.primary_interface.mac = Net::Validations.normalize_mac(values[:macaddress]) if values.present?
+        if values.present?
+          # bridge interfaces are not attached to parent interface so save would not be possible
+          if interface_class(identifier) != Nic::Bridge && values[:virtual].present? && values[:attached_to].present?
```

Comment:
I think we can just set values[:virtual] and values[:attached] to not looking at their values. Note that both bridge and bond don't use attached_to but rather attached_devices, therefore the check here is redundant. This could also make bond supported as primary interfaces. I suggest just dropping the ` && values[:virtual].present? && values[:attached_to].present?` from this condition